rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" category-id="optional">
+                 <hint title="Dynamic class instantiation" effort="1" category-id="optional">
```

Comment:
I am not sure what's the @Maarc 's current wish. Should it be 1 or 0? Do the PR's here represent the changes you want?